cuda_core/pyproject.toml

```
@@ -46,8 +46,8 @@ dependencies = [
 ]
 
 [project.optional-dependencies]
-cu11 = ["cuda-bindings==11.8.*"]
-cu12 = ["cuda-bindings==12.*"]
+cu11 = ["cuda-bindings[all]==11.8.*"]
+cu12 = ["cuda-bindings[all]==12.*"]
 test = ["cython>=3.0", "setuptools", "pytest>=6.2.4"]
 test-cu11 = ["cuda-core[test]", "cupy-cuda11x", "nvidia-cuda-runtime-cu11"]  # runtime headers needed by CuPy
```

btw I was testing something locally and got bitten by this
```suggestion
test-cu11 = ["cuda-core[cu11,test]", "cupy-cuda11x", "nvidia-cuda-runtime-cu11"]  # runtime headers needed by CuPy
```
(Same applies to test-cu12 below.)